[2024-09-22T01:20:58.479+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:20:50.577780+00:00 [queued]>
[2024-09-22T01:20:58.599+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:20:50.577780+00:00 [queued]>
[2024-09-22T01:20:58.600+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-09-22T01:20:58.726+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_cross_validation.RandomForestRegressor> on 2024-09-22 01:20:50.577780+00:00
[2024-09-22T01:20:58.786+0000] {standard_task_runner.py:60} INFO - Started process 5490 to run task
[2024-09-22T01:20:58.809+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_eval', 'model_cross_validation.RandomForestRegressor', 'manual__2024-09-22T01:20:50.577780+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpler5znpe']
[2024-09-22T01:20:58.810+0000] {standard_task_runner.py:88} INFO - Job 126: Subtask model_cross_validation.RandomForestRegressor
[2024-09-22T01:20:58.899+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-22T01:20:59.057+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_eval.model_cross_validation.RandomForestRegressor manual__2024-09-22T01:20:50.577780+00:00 [running]> on host cc45b92b61e2
[2024-09-22T01:20:59.729+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_eval' AIRFLOW_CTX_TASK_ID='model_cross_validation.RandomForestRegressor' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T01:20:50.577780+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T01:20:50.577780+00:00'
[2024-09-22T01:21:00.931+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-09-22T01:21:00.964+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_eval, task_id=model_cross_validation.RandomForestRegressor, execution_date=20240922T012050, start_date=20240922T012058, end_date=20240922T012100
[2024-09-22T01:21:01.106+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-09-22T01:21:01.167+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
